testing/tests/tests.py

```diff
@@ -126,7 +126,7 @@ def test_load_setting_abstract_method(self):
 
 class LoginRequiredMiddlewareTests(TestCase):
     def setUp(self):
-        self.login_url = reverse("django.contrib.auth.views.login")
+        self.login_url = reverse(django.contrib.auth.views.login)
```

Comment:
can we import this explicitly at the top and then just do 

`self.login_url = reverse(login)
`